src/fosslight_source/cli.py

```diff
@@ -19,6 +19,13 @@
 from .run_scancode import run_scan
 from .run_scanoss import run_scanoss_py
 
+extended_header = {}
```

Comment:
Let's add `SCANOSS_SHEET_NAME = 'SRC_FL_Source'`
And replace all `SRC_FL_Source `with `SCANOSS_SHEET_NAME`

Reply:
Done.